src/plugins/intel_cpu/tests/functional/test_utils/cpu_test_utils.cpp

```diff
@@ -140,6 +142,20 @@ void CPUTestsBase::CheckPluginRelatedResults(const ov::CompiledModel &execNet, c
     CheckPluginRelatedResults(execNet, std::set<std::string>{nodeType});
 }
 
+void CPUTestsBase::ExpectPluginRelatedResultsFailed(const ov::CompiledModel &execNet, const std::string& nodeType) {
```

Comment:
Instead of trying to catch the assert exception, why we don't just create a new instance of CPUSpecificParams?
You are currently using conv_avx512_2D_nspc_brgconv_amx. Can we just have something like:
conv_avx512_2D_nspc_amx_jit_fallback
and expect non-brgemm implementation.

Reply:
I mean thus we will avoid dealing with multithreading, gtests static EXPECT_FATAL_FAILURE limitations and other stuff.